src/createVariants.js

```diff
@@ -31,19 +30,21 @@ module.exports = function createVariants(baseConfig, variants, configCallback) {
     // Okay, so this looks a little bit messy but it really does make some sense.
     // Essentially, for each base configuration, we want to create every
     // possible combination of the configuration variants specified above.
-    var transforms = Object.keys(variants).map(function(key) {
+    const transforms = Object.keys(variants).map(function(key) {
             return function(config) {
                 return variants[key].map(function(value) {
-                    var result = assign({}, config);
+                    let result = assign({}, config);
```

Comment:
Since we're introducing babel for pre-compilation, we probably can rewrite this a bit:

```
const transforms = Object.keys(variants).map(key => config => variants[key].map(value => ({ ...config, [key]: value }));
```

Would allow us to get rid of the `lodash.assign` dependency.